lib/privatekey.js

```diff
@@ -157,14 +157,18 @@ PrivateKey._transformBuffer = function(buf, network) {
   }
 
   info.network = Networks.get(buf[0], 'privatekey');
```

Comment:
@fanatid The code already did the redundant define when I touched it, so I left it. Removing the line doesn't impact the tests, so it should be able to be removed safely.